tools/confluent-tools.sh

```diff
@@ -3,10 +3,18 @@
 # For commonly used command-line tools, this entrypoint script adds the
 # necessary parameters to point them at the linked containers.
 
+: ${TOOLS_ZK_CONNECTION_URL:=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT}
```

Comment:
We should name these to be clear. The only one that's a URL is `TOOLS_SR_CONNECTION_URL` -- the rest are just `<host>:<port>` strings (or multiple comma-separated `<host>:<port>` strings).

We have at least one instance of this mistake in CP -- `kafkastore.connection.url` isn't actually a URL -- but we should still try to make these accurate.